ted_sws/notice_transformer/adapters/rml_mapper.py

```diff
@@ -0,0 +1,48 @@
+import abc
+import pathlib
+import subprocess
+
+class RMLMapperABC(abc.ABC):
+    """
+        This class is a general interface of an adapter for rml-mapper.
+    """
+
+    @abc.abstractmethod
+    def execute(self, package_path: pathlib.Path) -> str:
+        """
+            This method allows you to perform an RML mapping based on a file package with a default structure.
+        :param package_path: path to package
+        :return: a string containing the result of the transformation
+        """
+
+
+class RMLMapper(RMLMapperABC):
+    """
+        This class is a concrete implementation of the rml-mapper adapter.
+    """
+
+    def __init__(self, rml_mapper_path: pathlib.Path):
+        """
+        :param rml_mapper_path: the path to the rml-mapper executable
+        """
+        self.rml_mapper_path = rml_mapper_path
+
+    def execute(self, package_path: pathlib.Path) -> str:
+        """
+            This method allows you to perform an RML mapping based on a file package with a default structure.
+            The package structure must be as follows:
+                /package_name
+                    /transform
+                        /mappings
+                            *.rml.tll
+
+                        /resources
+                            *.json
+                            ...
+                            *.xml
+        :param package_path: path to package
+        :return: a string containing the result of the transformation
+        """
+        bash_script = f"cd {package_path} && java -jar {self.rml_mapper_path} -m {package_path / 'transform/mappings/*'}"
+        script_result = subprocess.run(bash_script, shell= True, stdout=subprocess.PIPE)
+        return script_result.stdout.decode('utf-8')
```

Comment:
this output shall be logged with the default logger. Or ... . 

It is not clear what comes out of this?